ncbitax2lin.py

```diff
@@ -231,10 +232,14 @@ def generate_lineage_outputs(df, taxid_lineages_output_prefix, name_lineages_out
 
     logging.info('generating lineage-by-name output...')
     name_lineages_df = process_lineage_dd(name_lineages_dd)
-    write_output(name_lineages_output_prefix, "name lineages", name_lineages_df)
+    name_lineages_df.columns = name_lineages_df.columns.str.replace('no rank', 'no_rank')
+    write_output(name_lineages_output_prefix, "name lineages", name_lineages_df,
+                 ['tax_id', 'superkingdom', 'kingdom', 'phylum', 'class', 'order', 'family', 'genus', 'species'] +
+                 [col for col in name_lineages_df if col.startswith('no_rank')])
```

Comment:
Cool. You tested that with the additional columns the `update_lineagedb.rake` script still works, right?

Reply:
Yes, I tested.
As long as columns are added to the end of the CSV the other script works.